src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/ActiveConfiguredProjectsLoader.cs

```diff
@@ -57,43 +58,14 @@ protected override void Dispose(bool disposing)
 
         private async Task OnActiveConfigurationsChangedAsync(IProjectVersionedValue<IConfigurationGroup<ProjectConfiguration>> e)
         {
-            await GetLoadedProjectsAsync(e);
-        }
-
-        protected override IDisposable? LinkExternalInput(ITargetBlock<IProjectVersionedValue<IEnumerable<ConfiguredProject>>> targetBlock)
-        {
-            IReceivableSourceBlock<IProjectVersionedValue<IConfigurationGroup<ProjectConfiguration>>> sourceBlock = _activeConfigurationGroupService.ActiveConfigurationGroupSource.SourceBlock;
-
-            DisposableValue<ISourceBlock<IProjectVersionedValue<IEnumerable<ConfiguredProject>>>>? transformBlock = sourceBlock.TransformWithNoDelta(TransformAsync);
-
-            transformBlock.Value.LinkTo(targetBlock, DataflowOption.PropagateCompletion);
-
-            return transformBlock;
-        }
-
-        private async Task<IProjectVersionedValue<IEnumerable<ConfiguredProject>>> TransformAsync(IProjectVersionedValue<IConfigurationGroup<ProjectConfiguration>> projectVersionedValue)
-        {
-            List<ConfiguredProject> generatedResult = await GetLoadedProjectsAsync(projectVersionedValue);
-
-            return new ProjectVersionedValue<IEnumerable<ConfiguredProject>>(generatedResult, projectVersionedValue.DataSourceVersions);
-        }
-
-        private async Task<List<ConfiguredProject>> GetLoadedProjectsAsync(IProjectVersionedValue<IConfigurationGroup<ProjectConfiguration>> projectVersionedValue)
-        {
-            List<ConfiguredProject> generatedResult = new List<ConfiguredProject>();
-
-            foreach (ProjectConfiguration configuration in projectVersionedValue.Value)
+            foreach (ProjectConfiguration configuration in e.Value)
```

Comment:
@lifengl this `foreach` loads all configurations serially. Would you expect any benefit in running them concurrently?